### ISSUE TITLE
Chat's GALLERY/FILES/etc. item sorting is weird

### ISSUE THREAD

**Issue by iequidoo** [deltachat/deltachat-android#3304]
I.e. for "Last week", "This month", "Last month" it's "Recently sent/received", but for July 2024 etc. i see "Least recently sent/received" sorting. This makes sorting of old items irregular, i.e. i see an item from July 31 and it's followed by another one from June 1. Maybe just make the sorting as in Desktop i.e. w/o splitting into weeks/months and always "Recently sent/received"?

**Comment by adbenitez** [comment]
yeah I think it has also looked weird for me when I noticed time ago, would be nice to keep the newer-to-older order always for less confusion

**Comment by r10s** [comment]
the splitting is fine and useful, however, but indeed, sorting months older than "last month" in reverse order does not make sense and seems to be  just a bug

**Comment by iequidoo** [comment]
Yes, i'm fine with splitting. Just in case, for me it affects GALLERY and FILES, i added "/etc." to the issue because i guess it's the same for other media types, but i didn't check
